tests/integration/pilot/tunneling_test.go

```diff
@@ -120,15 +120,37 @@ func TestTunnelingOutboundTraffic(t *testing.T) {
 			externalForwardProxyIP := getPodIP(ctx, externalNs, "external-forward-proxy")
 
 			for _, proxyConfig := range forwardProxyConfigurations {
+				var egressNs string
+				var egressSvc string
+				var egressLabel string
+
+				if i.Settings().EgressGatewayServiceNamespace != "" {
```

Comment:
tests should not have to deal with this complexity (and won't - its guaranteed  someone will use `i.Settings().EgressGatewayServiceNamespace` on its own and break things silently). Just make `i.Settings().EgressGatewayServiceNamespace` default to `istio-system`

Reply:
same for the egress svc name and label too?

Reply:
yes

Reply:
The ingress and egress vars already say that they default to `istio-system` and the default labels, but I'm not sure where those defaults are set. https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/config.go#L171-L185

Would we need to set them here? https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/config.go#L78-L89

Reply:
Also, some tests (ex: https://github.com/istio/istio/blob/master/tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go#L235) don't have `istio-egressgateway` as the default istio label.

Reply:
Nvm, I guess the default label is usually `egressgateway`, whereas the default Service name is `istio-egressgateway`?